### DISCUSSION TITLE
🚀 你的选择，定义明天的 xray！ 🚀

### DISCUSSION THREAD

**Discussion by Jarcis-cy** [chaitin/xray#1804] [Polls]
🌟 各位开发者、xray 的粉丝们，注意啦！在这个代码构建的时代，每一款得心应手的工具都将成为你征服技术巅峰的利器。而现在，是时候让我们听到你的声音了！

### 🎉 活动亮点：

- **你来选，我们来实现！** 我们精心准备了一系列令人兴奋的 xray 2.0 工具预选项，覆盖各个领域，只等你的一票决定哪一个将优先进入开发日程！
- **长期调研，持续影响！** 这不仅仅是一次投票，而是一项长期的调研计划。你的每一次参与，都将持续影响社区未来的发展方向，让我们的技术生态更加贴近每一位开发者的心声。

🚀 你的选择将塑造明天的 xray！加入我们，让你的声音被听到！

---

### 🚀 Your Choice Defines the Future of xray! 🚀

🌟 Attention all developers and fans of xray! In this era of building with code, every handy tool becomes a powerful weapon in your quest to conquer the heights of technology. Now, it's time for us to hear your voice!

### 🎉 Event Highlights:

- **You Choose, We Implement!** We have meticulously prepared a series of exciting pre-selection options for the xray 2.0 toolset, covering various fields. Your vote will decide which one gets prioritized in our development schedule!
- **Long-term Survey, Lasting Impact!** This is not just a one-time vote; it's a long-term research plan. Every time you participate, you continuously influence the future direction of the community, making our tech ecosystem more in tune with the desires of every developer.

🚀 Your choice will shape tomorrow's xray! Join us and let your voice be heard!

**Comment by yywing** [comment]
针对某通用漏洞的扫描工具

大家期待的通用漏洞类型有哪些？可以在这个评论下回复

**Reply by Niiiiko** [reply]
log4j、fj、sql、ssrf、目录便利，spring actuator、druid、swagger
以及通用漏洞扫描是否支持被动扫描模式

**Reply by cri1wa** [reply]
等一个被动扫描
等一个被动扫描
等一个被动扫描

**Comment by yywing** [comment]
针对某产品的专项扫描工具
大家期待的支持产品有哪些？可以在这个评论下回复

**Comment by safebaseline** [comment]
没有xray我怎么投

**Comment by safebaseline** [comment]
已收到

**Comment by JoJenH** [comment]
复活吧 我的被动扫描

**Comment by safebaseline** [comment]
已收到
